Home.md

```diff
@@ -6,7 +6,7 @@ On the right side of the page you see the available wiki pages. For new users, y
 
 If you encounter a bug, please goto [Discord](https://discord.com/invite/U8WPBdf) and open a Chat, Help or Issue.
 
-These documents are a work-in-progress. If you'd like to contribute, please get in touch with us on [Discord](https://discord.com/invite/U8WPBdf), or [Discussions](https://discord.com/channels/502612977271439372/502612977803984898), [Telegram](https://t.me/sickchill), [IRC](https://kiwiirc.com/client/irc.freenode.net/?theme=basic#sickchill) or open an [issue here on GitHub.](https://github.com/SickChill/SickChill/issues)
+These documents are a work-in-progress. If you'd like to contribute, please get in touch with us on Discord in [Discussions](https://discord.com/channels/502612977271439372/502612977803984898), or open an [issue here on GitHub.](https://github.com/SickChill/SickChill/issues)
```

Comment:
The part about opening an issue on GitHub can go away, we do issues on discord now

Reply:
Also, I think this was the part you could put both a link to the discord invite and the discussion. Two links, one with Discussions and the other with (Invite) or something.

Reply:
> The part about opening an issue on GitHub can go away, we do issues on discord now

Done.  Might need to look for similar references elsewhere in the wiki.

> Also, I think this was the part you could put both a link to the discord invite and the discussion. Two links, one with Discussions and the other with (Invite) or something.

Done.  Took a stab at that.

Reply:
Found additional references and tried to clean that up as well.